final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,125 @@
+import socket
+import sys
+import requests
+import argparse
+import json
+import logging
+import pycodestyle
```

Comment:
pycodestyle is a tool so you do not actually need to import it anywhere. If you want to use it just type this in the command line: ```pycodestyle <file_name>.py``` and it will analyse it.

The same for requirements.txt file.